website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -123,7 +123,7 @@ Eg. `"nodejs10"`, `"nodejs12"`, `"nodejs14"`, `"python37"`, `"python38"`, `"pyth
 
 * `event_trigger` - (Optional) A source that fires events in response to a condition in another service. Structure is documented below. Cannot be used with `trigger_http`.
 
-* `trigger_http` - (Optional) Boolean variable. Any HTTP request (of a supported type) to the endpoint will trigger function execution. Supported HTTP request types are: POST, PUT, GET, DELETE, and OPTIONS. Endpoint is returned as `https_trigger_url`. Cannot be used with `trigger_bucket` and `trigger_topic`.
+* `trigger_http` - (Optional) Boolean variable. Any HTTP request (of a supported type) to the endpoint will trigger function execution. Supported HTTP request types are: POST, PUT, GET, DELETE, and OPTIONS. Endpoint is returned as `https_trigger_url`. Use `event_trigger` for another trigger options.
```

Comment:
Do we want to make this "Cannot be used with `event_trigger`." to stay consistent with the messaging of `event_trigger`?

Reply:
Hi @megan07 ,
Thank you for reviewing this PR. I updated the PR per your suggestion to be consistent with messaging in `event_trigger`. Thank you.